### DISCUSSION TITLE
Contributing

### DISCUSSION THREAD

**Discussion by TashaSkyUp** [CombatExtended-Continued/CombatExtended#815] [General]
How do I contribute please?

**Comment by N7Huntsman** [comment]
Sorry for the long delay getting back to you, we primarily work off of our Discord. The best way to get ahold of us would be from there (https://discord.gg/D3szS78pHW). In terms to contributing, that'd depend entirely upon what you're interested in working on. The project includes everything from C# development and .xml work to playtesting and artwork.
